### PR TITLE
Fix support for travis pro

### PR DESCRIPTION
**Pull request by florianmutter**
With the change from #128 travis-pro stopped working. This re-adds support for it again and updates the readme.

### CODE REVIEW

**Comment by florianmutter** [comment]
FYI: I did not try this yet. I still need to do so

**Comment by florianmutter** [comment]
I tested it and it works

File: README.md [hunk 1]
```diff
@@ -161,12 +161,13 @@ scalacOptions += Seq("-encoding", "UTF-8")
...
 
-It is important to set the correct `service_name` when using Travis-Pro.  The default is to use `travis-ci`.  To override this value, add the following to your `build.sbt`
+It is important to set the correct `service` when using Travis-Pro.  The default is to use `travis-ci`.  To override this value, add the following to your `build.sbt`
 
 ```scala
 import org.scoverage.coveralls.Imports.CoverallsKeys._
+import org.scoverage.coveralls.TravisPro
 
-coverallsServiceName := Some("travis-pro")
+coverallsService := Some(TravisPro)
```
Comment (ckipp01) [inline README.md:170]: 👍🏼 thanks for catching this.

**Review by ckipp01 — APPROVED** [review]
Looks good, thanks!

**Comment by florianmutter** [comment]
Thank you very much for merging it. Would it be possible to release this change? This would allow us to merge our scala steward update pr on sbt-coveralls :)

**Comment by ckipp01** [comment]
> Thank you very much for merging it. Would it be possible to release this change? This would allow us to merge our scala steward update pr on sbt-coveralls :)

Just pushed a tag. You should see it in an hour or so as soon as it all propagates.

**Pull request merged**